### DISCUSSION TITLE
I want to switch from Revi OS to AtlasOS

### DISCUSSION THREAD

**Discussion by rafpzsx** [Atlas-OS/Atlas#1222] [Help & Support]
I used to have atlas os, but i wanted to try Revi OS, turns out Revi OS performed worse than Atlas OS, and i wanted to go back to Atlas OS, can i still use AME Wizard and switch to Atlas OS or i have to full reinstall?

**Comment by H0ggu** [comment]
Full reinstall, 
Follow the docs from start to finish and you'll be fine
https://docs.atlasos.net/getting-started/installation/
